Makefile

```diff
@@ -87,9 +87,11 @@ install: uninstall
 uninstall:
 	@test -f "$(DEST)/bin/lima" || echo "lima not found in $(DEST) prefix"
 	rm -rf \
+		"$(DEST)/bin/docker.lima" \
```

Comment:
Nit: I’d prefer this to be between nerdctl.lima and podman.lima.

Reply:
I had it at first before `nerdctl.lima`, so the 3 `*.lima` names were sorted alphabetically, but then force-pushed it to make all name sorted correctly. So you want this:

```
	rm -rf \
		"$(DEST)/bin/lima" \
		"$(DEST)/bin/limactl" \
		"$(DEST)/bin/nerdctl.lima" \
		"$(DEST)/bin/docker.lima" \
		"$(DEST)/bin/podman.lima" \
		"$(DEST)/share/lima" "$(DEST)/share/doc/lima"
```

Reply:
Yes, because docker.lima and podman.lima are kinda `/contrib` files.